python/lsst/rubintv/production/starTracker.py

```diff
@@ -755,10 +818,10 @@ def getMissingImageSeqNums(self, camera):
             # camera then the column won't exist and the process thrashes a bit
             return []
 
-        missing = [s for s in seqNums if mdTable[successfulFitColumn][s] is np.nan]
+        missing = [s for s in seqNums if mdTable[successfulFitColumn][s] is not np.nan]
```

Comment:
Ah, it's because we flipped the logic - `getMissingImageSeqNums` is now `getFullyProcessedSeqNums` so you're absolutely right, it's just that the function's functionality is now reversed 🙂

Reply:
That makes sense. Please rename this variable from `missing` to something like `successful`

Reply:
lol, right, good point 😄